scripts/build_ssg_content.sh

```diff
@@ -51,14 +51,14 @@ cd build
 # Need to build content to support SCAP 1.2
 # more info here: https://github.com/ComplianceAsCode/content/blob/master/docs/manual/developer_guide.adoc#331-building-compliant-scap-12-content
 
-cmake -DSSG_TARGET_OVAL_MINOR_VERSION:STRING=10 ../
+cmake -DSSG_TARGET_OVAL_MINOR_VERSION:STRING=11 ../
 
 # Build RHEL first to generate dependencies for CentOS and Oracle Linux
 make -j4 "${MAKE_TARGETS_RHEL[@]}"
 make -j4 "${MAKE_TARGETS_OTHERS[@]}"
 
-find . -type f -name '*ds-1.2*' -exec rename ds-1.2.xml ds.xml '{}' \;
-find . -type f -name '*xccdf-1.2*' -exec rename xccdf-1.2.xml xccdf.xml '{}' \;
+find . -type f -name '*ds-1.2*' -exec rename ds-1.3.xml ds.xml '{}' \;
+find . -type f -name '*xccdf-1.2*' -exec rename xccdf-1.3.xml xccdf.xml '{}' \;
 
 cp *-ds.xml *-xccdf.xml *-oval.xml *-cpe-dictionary.xml "$DIST_DIR"
```

Comment:
Ok, reviewing the built files, I think we can skip the rename step entirely, and just grab all the `ssg-*.xml` files.
```suggestion

cp ssg-*.xml "$DIST_DIR"
```